### PR TITLE
Selenium screenshot container

### PR DESCRIPTION
**Pull request by ssaldarriaga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+
+## Allowed arguments
+1. **url**: the URL of the page to take the screenshot
+2. **file_name**: Name of the image which will be saved (e.g `/path/of/my/image.png`)
+3. **width**: Initial width for the browser
+4. **height**: Initial height for the browser
+5. **app_selector_xpath** (*optional*): XPath selector for search the target content in the DOM. Default `//div[@id='root']`
+6. **not_load_selector_xpath** (*optional*): XPath selector when it can't find the content. Default `div[contains(@class, 'selenium-data-not-loaded')]`
+
+**Note**: Everything arguments are positional, each one must be sent in the order described above.
```
Comment (d4vsanchez) [inline README.md:14]: There's a typo here, it should say: "Every argument is positional" or "Each one of the arguments are positional".

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+
+## Usage
+To build the container, execute:
+```
+# Switch to Dockerfile path
+cd docker-selenium-screenshot
+
+# Run docker build
+docker build docker build --tag <tag name> .
```
Comment (d4vsanchez) [inline README.md:24]: This command seems to have `docker build` duplicated.

File: src/selenium_screenshot.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        chrome_options.add_experimental_option("excludeSwitches", ["enable-automation"])
+        chrome_options.add_experimental_option(
+            "prefs", {"download.prompt_for_download": False}
+        )
+        browser = webdriver.Chrome(options=chrome_options)
+        browser.set_window_size(window_width, window_height)
+        browser.get(url)
+
+        return browser
+    except Exception as ex:
```
Comment (d4vsanchez) [inline src/selenium_screenshot.py:37]: Isn't this a violation of a pylint rule? The `broad-except` rule

File: src/selenium_screenshot.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+        # Resize window for canvas widgets
+        resize_chromium_viewport(browser, height)
+
+        # Wait a little bit for DOM changes
+        time.sleep(2)
+
+        # Take the screenshot
+        _take_screenshot(browser, file_name)
+    except Exception as ex:
```
Comment (d4vsanchez) [inline src/selenium_screenshot.py:113]: Same here, isn't this a violation of the `broad-except` rule?

File: src/selenium_screenshot.py (hunk 3 above)
Comment (ssaldarriaga) [reply src/selenium_screenshot.py:37]: Yes, it is. But I don't know and I'm not sure which errors will be throw, then I preferred to use `Exception`.

File: src/selenium_screenshot.py (hunk 4 above)
Comment (ssaldarriaga) [reply src/selenium_screenshot.py:113]: Yes, it is. But I don't know and I'm not sure which errors will be throw, then I preferred to use `Exception`.

**Pull request merged**